File: back/src/main/java/org/lendi/umtapo/entity/Address.java
package org.lendi.umtapo.entity;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

/**
 * Address entity.
 *
 * Created by axel on 29/11/16.
 */
@Entity
@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
public class Address {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;
    private String address1;
    private String address2;
    private String zip;
    private String city;
    private String phone;
    private String email;

    public Address() {
    }

    public Address(String address1, String address2, String zip, String city, String phone, String email) {
        this.address1 = address1;
        this.address2 = address2;
        this.zip = zip;
        this.city = city;
        this.phone = phone;
        this.email = email;
    }

    public Integer getId() {
        return id;
    }


    public void setId(Integer id) {
        this.id = id;
    }


    public String getAddress1() {
        return address1;
    }


    public void setAddress1(String address1) {
        this.address1 = address1;
    }


    public String getAddress2() {
        return address2;
    }


    public void setAddress2(String address2) {
        this.address2 = address2;
    }


    public String getZip() {
        return zip;
    }


    public void setZip(String zip) {
        this.zip = zip;
    }


    public String getCity() {
        return city;
    }


    public void setCity(String city) {
        this.city = city;
    }


    public String getPhone() {
        return phone;
    }


    public void setPhone(String phone) {
        this.phone = phone;
    }


    public String getEmail() {
        return email;
    }


    public void setEmail(String email) {
        this.email = email;
    }
}

File: front/src/entity/address.ts
export class Address {
    private id: number;
    private address1: string;
    private address2: string;
    private zip: string;
    private city: string;
    private phone: string;
    private email: string;


    getId(): number {
        return this.id;
    }

    setId(value: number) {
        this.id = value;
    }

    getAddress1(): string {
        return this.address1;
    }

    setAddress1(value: string) {
        this.address1 = value;
    }

    getAddress2(): string {
        return this.address2;
    }

    setAddress2(value: string) {
        this.address2 = value;
    }

    getZip(): string {
        return this.zip;
    }

    setZip(value: string) {
        this.zip = value;
    }

    getCity(): string {
        return this.city;
    }

    setCity(value: string) {
        this.city = value;
    }

    getPhone(): string {
        return this.phone;
    }

    setPhone(value: string) {
        this.phone = value;
    }

    getEmail() {
        return this.email;
    }

    setEmail(value) {
        this.email = value;
    }
}
